src/main/java/network/brightspots/rcv/CastVoteRecord.java

```diff
@@ -128,6 +128,12 @@ boolean doesUseLastAllowedRanking() {
     return usesLastAllowedRanking;
   }
 
+  // This represents the canonical ID used for audit logs and RCTab CVR
+  String getPrimaryId() {
+    return !isNullOrBlank(computedId) ? computedId : suppliedId;
+  }
+
+  // TODO: Before merging PR determine if we still need getId & the Record ID it prints to the RCTab CVR
   String getId() {
     return suppliedId != null ? suppliedId : computedId;
   }
```

Comment:
This is the logic that is used to generate the "Record Id" that gets printed out in the RCTab CVR. It is not clear to me if this was intended to be a unique CVR identifier, because it uses the same `computedId` and `suppliedId` that my `getPrimaryId` uses.

But this value returned by `getId()` is not unique to every CVR (tested on 2024 Alaska US House race).

Reply:
@yezr thinks we don't need this any more.

Reply:
Either fix this with the new logic or remove this method.

Reply:
Need to fix this line:
`csvPrinter.print(castVoteRecord.getId());`
in `OutputWriter.java`

Reply:
Move the code in here to a `getSuppliedId` method and what is "Record ID" in the RCTab CVR should be "Vendor Id"